modules/nodes-pods-autoscaling-custom-prometheus-config.adoc

```diff
@@ -0,0 +1,175 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-custom.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-autoscaling-custom-prometheus-config_{context}"]
+= Configuring the custom metrics autoscaler to use {product-title} monitoring
+
+You can use the installed {product-title} monitoring as a source for the metrics used by the custom metric autoscaler. However, there are some additional configurations you must perform.
+
+[NOTE]
+====
+These steps are not required for an external Prometheus source.
+====
+
+You must perform the following tasks, as described in this section:
+
+* create a service account to get a token
+* create a role
+* add that role to the service account
+* reference the token in the trigger authentication object used by Prometheus
+
+.Prerequisites
+
+* {product-title} monitoring must be installed.
+
+* The Custom Metrics Autoscaler Operator must be installed. 
+
+.Procedure
+
+To configure custom metrics autoscaler to work with Prometheus Custom Monitoring:
+
+. Change to the project with the the object you want to scale:
++
+[source,terminal]
+----
+$ oc project my-project
+----
+
+. Use the following command to create a service account, if your cluster does not have one:
++
+[source,terminal]
+----
+$ oc create serviceaccount <service_account>
+----
++
+where:
++
+<service_account>:: Specifies the name of the service account.
+
+. Use the following command to locate the token assigned to the service account:
++
+[source,terminal]
+----
+$ oc describe serviceaccount <service_account>
+----
++
+where:
++
+<service_account>:: Specifies the name of the service account.
++
+.Example output
```

Comment:
I think this example output is indented too far. It's indented with the service_account description, but it should be left aligned with the command block above it

Reply:
@bergerhoffer Do you know how to configure this in asciidoc?

Reply:
@mburke5678 I usually have to play with things like this to see what works.

I think if you put a blank line above the + above the example output, it shifts the indentation correctly.

If that doesn't work, I think you can also use `--` around things to keep the indentation right. Like this might work:

```
[source,terminal]
----
$ oc describe serviceaccount <service_account>
----
+
--
where:

<service_account>:: Specifies the name of the service account.
--
+
.Example output
[source,terminal]
```